test_communication/test/test_publisher_subscriber.py.in

```diff
@@ -6,14 +8,20 @@ from launch.launcher import DefaultLauncher
 def test_publisher_subscriber():
     ld = LaunchDescriptor()
 
+    publisher_env = dict(os.environ)
+    publisher_env['RCL_ASSERT_RMW_ID_MATCHES'] = '@PUBLISHER_RMW@'
```

Comment:
This line is currently ineffective since the `env` parameter is not used. So the expected test is not happening atm.